www/js/app.run.js

```diff
@@ -15,17 +15,32 @@ function runBlock ($ionicPlatform, $location, $ionicHistory) {
     if(window.StatusBar) {
       StatusBar.styleDefault();
     }
+
+    SoundServices.setIsActive(false);
+    if(SoundServices.getIsActive()){
+      SoundServices.preloadSimple('intro', 'audio/intro.wav');
```

Comment:
it should be stored in a dictionary where we can adjust later without touching codebase